.github/workflows/test-labs.yaml

```
@@ -0,0 +1,78 @@
+name: Test
+on:
+  workflow_dispatch:
+    inputs:
+      kube-distro:
+        type: choice
+        description: "The type of kubernetes distribution to use, i.e minikube, kind"
+        options:
+          - minikube
+          - kind
+        default: kind
+        required: false
+
+      istio-ready:
+        type: boolean
+        description: "Set to true if istio can be tested, leave false otherwise"
+        required: false
+
+      kubernetes-tests:
+        type: string
+        description: "The name(s) of the kubernetes test(s) to be run, if you want to run multiple, provide them separated with spaces, e.g 'kvstore splitdim', if you provide 'all', it will execute all kubernetes tests"
```

We have three cases if I recall correctly: `"helloworld" "splitdim", "kvstore"` I would put a phrasing like this:
```suggestion
        description: "Kubernetes tests to be run. Options: 'helloworld' 'splitdim', 'kvstore', 'all'.  if you want to run multiple, provide them separated with spaces (e.g., 'kvstore splitdim'); 'all' means execute all tests"
```